src/client/components/SignInButton/Button.js

```diff
@@ -8,13 +8,15 @@ const Button = (props) => {
     onclick();
   };
   return (
-    <button type="submit" className="button" onClick={clicked}>
-      <img
-        alt="Sign in with Slack"
-        src="https://platform.slack-edge.com/img/sign_in_with_slack.png"
-        srcSet="https://platform.slack-edge.com/img/sign_in_with_slack.png 1x, https://platform.slack-edge.com/img/sign_in_with_slack@2x.png 2x"
-      />
-    </button>
+    <div className="box">
```

Comment:
`box` is a very generic name and you already are using this class in the footer. Which means that `.box` will affect it.
Also, a `button` shouldn't style its location and/or placement - that is up to the container _around_ it.